src/views/AddItem.jsx

```diff
@@ -29,12 +29,34 @@ export function AddItem({ listToken }) {
 		}, 3000);
 	};
 
+	const filtered = data.filter((item) =>
```

Comment:
I would suggest that you do that filtering inside the handleSubmit function so that this variable doesn't get computed each time itemData changes but only once when you actually do the check :)